CMakeLists.txt

```diff
@@ -144,22 +35,39 @@ if(BUILD_EXAMPLES)
     add_subdirectory(example)
 endif()
 
-# Workaround config file with crypto3::all as a dependency
-# TODO: remove after resolving dependency issues in crypto3
+# Install phase
+
+set(EXPORT_NAME crypto3_blueprint-targets)
+# Install target
+install(TARGETS blueprint EXPORT ${EXPORT_NAME})
+# Install headers
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+# Generate and install package config files
```

Comment:
This is probably the most controversial part of the changes. I added a bunch of cmake library functions (`configure_package_config_file`, `write_basic_config_version_file`, `install(FILES ...)`, `install(EXPORT...)`) plus `Config.cmake.in` file instead of single `cm_deploy` call. I still think this is a better solution. Here are some arguments:
- Much easier to debug. All arguments of cmake functions are in front of eyes and defined in the very same file. Comparing to case when Boost modules set some target properties in one function and then extract them in another one to pass inside `install`, so you need to examine whole cmake trace to understand what is going on, the proposed solution is very simple to debug.
- It is more flexible. If we would need to export some option of blueprint, like `BLUEPRINT_PLACEHOLDER_PROOF_GEN`, there is no way to do it with `cm_deploy` without rewriting a bunch of cmake modules. That is because `cm_deploy` generates config file from a unified template. But here we can simply add it to `Config.cmake.in` file.

I would agree that `cm_deploy` is the best option when we had 20+ repositories with complex graph of dependencies. But since we reduced the number to a couple, manual config generation seems preferable. If we want to reduce an amount of code to a minimum, I think we could create some simplified analogue of `cm_deploy` function that would include the commands below.